modules/impact.base.game.d.ts

```diff
@@ -42,20 +42,34 @@ declare global {
         windowFocusChanged: Addons.EventHandlersArr<'onWindowFocusChanged'>;
       }
 
-      type TeleportLoadHint = 'NEW' | 'LOAD' | null | undefined;
+      type TeleportLoadHint = Optional<'NEW' | 'LOAD'>;
     }
     interface Game extends ig.Class {
-      paused: boolean;
       entities: ig.Entity[];
       physics: ig.Physics;
       playerEntity: ig.ENTITY.Player;
       addons: Game.Addons;
+      mapName: string;
+      paused: boolean;
+      events: ig.EventManager;
```

Comment:
Misordered properties
```suggestion
      paused: boolean;
      mapName: string;
      entities: ig.Entity[];
      events: ig.EventManager;
      physics: ig.Physics;
      playerEntity: ig.ENTITY.Player;
      addons: Game.Addons;
```